ros_buildfarm/devel_job.py

```diff
@@ -198,6 +200,9 @@ def configure_devel_job(
     if build_file is None:
         build_files = get_source_build_files(config, rosdistro_name)
         build_file = build_files[source_build_name]
+    # Overwrite build_file.targets if build_targets is specified
+    if not build_targets is None:
```

Comment:
A slight style note: PEP8 calls for 'is not None' it's functionally the same but cleaner to read.